storage/blog/live-from-godotcon-boston-web-net-prototype/slides_state-of-the-godot-and-the-web-2025/index.html

```diff
@@ -0,0 +1,1731 @@
+<!DOCTYPE html>
```

Comment:
That's a lot of code and data/fonts for a slick deck that's only linked in a blog post. Is this the framework you built your presentation with or did you convert it all to HTML just for the blog post? Because otherwise a PDF would likely do just fine.

Reply:
It's actually the exported slides from [slides.com](slides.com) that uses [the open source framework reveal.js](https://revealjs.com/) internally. There is some sound effects/video that would be lost though if not included.

I thought linking directly to the public url for my slides on slides.com, but I thought that hosting the files directly would maybe prevent some link rot.

Reply:
Though, I could remove unused fonts. It seems that it tries to package fonts that I didn't even use.

Reply:
Is it possible to view this slideshow (with sound effects/video) somehow, without having to build the site locally?

Reply:
Did remove unused fonts, and it work flawlessly. The number of files this PR was dramatically reduced.

I just tried to export the slides to PDF and it's all broken, unfortunately.